airbyte-integrations/connectors/source-woocommerce/source_woocommerce/manifest.yaml

```diff
@@ -36,7 +36,7 @@ definitions:
     end_time_option:
       field_name: modified_before
       inject_into: request_parameter
-    cursor_field: "{{ parameters['stream_cursor_field'] }}"
+    cursor_field: "date_modified_gmt"
```

Comment:
Some streams used the same `cursor_field` but used interpolation to reference `stream_cursor_field`, I removed all instances and instead defined it directly the stream slicer where it should belong